test/index.test.js

```diff
@@ -153,3 +153,14 @@ describe('trim option', () => {
     );
   });
 });
+
+describe('corrupted styles', () => {
```

Comment:
Do you mind renaming `corrupted` to `invalid`?

Reply:
Sorry. Just saw this comment. Will redo PR now.

Reply:
Thanks!